src/components/Users.jsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react";
+import UserItem from "./UserItem";
+
+export default function Users() {
+  const [users, setUsers] = useState([]);
+  useEffect(() => {
+    fetch("https://random-data-api.com/api/users/random_user?size=10")
+      .then((res) => res.json())
+      .then((data) => {
+        setUsers(data);
+      });
+  }, []);
+  return (
+    <div className="w-full h-full flex justify-center flex-wrap gap-5 bg-[#E1F6DC] py-10">
```

Comment:
You are missing a button here, and also I would suggest that you write the API fetching logic inside a function because you will need it on the click event. 
![image](https://user-images.githubusercontent.com/56281095/223123534-0a229da4-8033-4578-a65d-a81d8e7c72e4.png)